src/core/contrib.js

```diff
@@ -71,22 +74,49 @@ export async function run(conf) {
     return;
   }
   const indexes = await response.json();
-  const { issues_url, issue_comment_url, contributors_url } = indexes;
+  const {
+    issues_url,
+    issue_comment_url,
+    comments_url,
+    contributors_url,
+  } = indexes;
+
+  const [
+    issues,
+    issueComments,
+    otherComments,
+    contributors,
+  ] = await Promise.all(
+    [issues_url, issue_comment_url, comments_url, contributors_url].map(url =>
+      fetchAll(
+        new URL(url.replace(/\{[^}]+\}/, ""), window.location.origin).href,
```

Comment:
Do the `url.replace(/\{[^}]+\}/, "")` into a variable. Then use that variable. when building the URL.